cnd/src/network/comit_ln.rs

```diff
@@ -129,6 +129,9 @@ impl ComitLN {
         Ok(())
     }
 
+    // TODO: This function needs to get its data from the database _not_ from
```

Comment:
There is a ticket for this: https://github.com/comit-network/comit-rs/issues/2546

Reply:
yeah, I'd remove that comment.

Reply:
Woops this was added before i decided to do this as a separate PR :) will remove all the TODOs.